src/System.Management.Automation/engine/CmdletParameterBinderController.cs

```diff
@@ -1690,8 +1690,8 @@ private void HandleRemainingArguments()
                         //     Set-ClusterOwnerNode -Owners foo,bar
                         //     Set-ClusterOwnerNode foo bar
                         //     Set-ClusterOwnerNode foo,bar
-                        // we unwrap our List, but only if there is a single argument of type object[].
-                        if (valueFromRemainingArguments.Count == 1 && valueFromRemainingArguments[0] is object[])
+                        // we unwrap our List, but only if there is a single argument which is a collection.
+                        if (valueFromRemainingArguments.Count == 1 && null != LanguagePrimitives.GetEnumerable(valueFromRemainingArguments[0]))
```

Comment:
I suggest using `LanguagePrimitives.IsTypeEnumerable(Type)` for 2 reasons:
1. `LanguagePrimitives.GetEnumerable` unwrap PSObject, which might strip off ETS properties associated with the PSObject.
2. `LanguagePrimitives.GetEnumerable` actually calls the `GetEnumerable` delegate which may throw an exception or cause a long delay (e.g. fetching object from Azure).

Reply:
Will do